pkg/okctl/okctl.go

```diff
@@ -778,7 +781,7 @@ https://www.passwordstore.org/
 		github.NewAuthDeviceFlow(github.DefaultGithubOauthClientID, github.RequiredScopes()),
 	)
 
-	o.CredentialsProvider = credentials.New(aws.New(authStore, saml), gh)
+	o.CredentialsProvider = credentials.New(aws.New(authStore, envRetriever, samlRetriever), gh)
```

Comment:
Since `envRetriever` only "responds" to `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` I think this will work. At least it decreases the likelihood that the user is surprised.

I know you aren't a huge fan of `NoInput`, which I understand. So I am  wondering if we should make the configuration of credentials more explicit in some other way? Perhaps we have a `--credentials-type` field with default set to `SAML` or something to that effect. Then we can add a second type `accesskey` or something? What do you think?